verifier/src/report_body.rs

```diff
@@ -110,6 +120,15 @@ impl<T> GreaterThanEqualVerifier<T> {
     }
 }
 
+impl<T> Display for GreaterThanEqualVerifier<T>
+where
+    T: SpacedStructName + Display,
+{
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+        write!(f, "The {} is {}", T::spaced_struct_name(), self.expected)
```

Comment:
:wrench: Similar to above this should probably be something like
```suggestion
        write!(f, "The {} should be greater than or equal to {}", T::spaced_struct_name(), self.expected)
```

Reply:
Done.